tests/CompilesIconsTest.php

```diff
@@ -13,7 +14,7 @@ class CompilesIconsTest extends TestCase
     /** @test */
     public function it_compiles_a_single_anonymous_component()
     {
-        $result = svg('healthicon-n-ambulatory_clinic')->toHtml();
+        $result = svg('healthicon_n-ambulatory_clinic')->toHtml();
```

Comment:
As mentioned this is the main change when using multiple icon sets for a single icon pack.
The prefix for each set must be unique and cannot use a `-` in it. So unfortunately we can't use `healthicon-n` as the prefix for the negative type icon set - instead we're using `healthicon_n`.

A potential alternative for this could be to just suffix the prefix base in the config with a single letter. In my blade box icons my packs are `bxs`, bxl` for instance. So we could adjust things so each pack here would be: `hin`, `his`, and `hio`. Where users could customize the `hi` part via the config.

Reply:
@mallardduck Now I get the difference in your icon pack, I agree we should go this way